common/persistence/src/main/resources/trl-derivation.yaml

```diff
@@ -0,0 +1,10 @@
+trl:
+  trl-mapping:
+    1: 1
+    2: 2
+    3: 3
+    4: 4
+    5: 5
+    6: 6
+    7: 7
+    8: 8
```

Comment:
@mlenkeit is the identity correct for now?

Reply:
Yes, according to spec (see [here](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/rapid-test/docs/spec/corona-test-server.md#submission-trl-mapping)).

> With the default mapping, Diagnosis Keys are stored as submitted.